crates/trippy-tui/locales/app.yml

```diff
@@ -3,439 +3,549 @@ trippy:
   en: "trippy"
   fr: "trippy"
   tr: "trippy"
+  zh: "trippy"
 auto:
   en: "auto"
   fr: "automatique"
   tr: "otomatik"
+  zh: "自动"
 on:
   en: "on"
   fr: "activé"
   tr: "açık"
+  zh: "开"
 off:
   en: "off"
   fr: "désactivé"
   tr: "kapalı"
+  zh: "关"
 yes:
   en: "Yes"
   fr: "Oui"
   tr: "Evet"
+  zh: "是"
 no:
   en: "No"
   fr: "Non"
   tr: "Hayır"
+  zh: "否"
 none:
   en: "none"
   fr: "aucun"
   tr: "hiçbiri"
+  zh: "无"
 hidden:
   en: "Hidden"
   fr: "Caché"
   tr: "Gizli"
+  zh: "隐藏"
 flow:
   en: "flow"
   fr: "flux"
   tr: "akış"
+  zh: "流量"
 flows:
   en: "flows"
   fr: "flux"
   tr: "akışlar"
+  zh: "流量"
 target:
   en: "Target"
   fr: "Cible"
   tr: "Hedef"
+  zh: "目标"
 config:
   en: "Config"
   fr: "Config"
   tr: "Konfigurasyon"
+  zh: "配置"
 status:
   en: "Status"
   fr: "Statut"
   tr: "Durum"
+  zh: "状态"
 protocol:
   en: "protocol"
   fr: "protocole"
   tr: "protokol"
+  zh: "协议"
 as-info:
   en: "as-info"
   fr: "info-as"
   tr: "AS bilgisi"
+  zh: "AS-信息"
 details:
   en: "details"
   fr: "détails"
   tr: "ayrıntılar"
+  zh: "详情"
 max-hosts:
   en: "max-hosts"
   fr: "nombre d'hôtes maximum"
   tr: "maksimum hostlar"
+  zh: "最大主机数"
 privacy:
   en: "privacy"
   fr: "confidentialité"
   tr: "gizlilik"
+  zh: "隐私"
 privileged:
   en: "privileged"
   fr: "privilégié"
   tr: "ayrıcalıklı"
+  zh: "特权"
 unprivileged:
   en: "unprivileged"
   fr: "non privilégié"
   tr: "ayrıcalıksız"
+  zh: "非特权"
 na:
   en: "n/a"
   fr: "non disponible"
   tr: "yok"
+  zh: "无"
 discovered:
   en: "discovered %{hop_count} hops"
   fr: "%{hop_count} sauts découverts"
   tr: "%{hop_count} atlanan keşfedildi"
+  zh: "发现 %{hop_count} 跳"
 discovered_flows:
   en: "discovered %{hop_count} hops and %{flow_count} unique %{plural_flows}"
   fr: "%{hop_count} sauts et %{flow_count} %{plural_flows} uniques découverts"
   tr: "%{hop_count} atlanan ve %{flow_count} benzersiz %{plural_flows} keşfedildi"
+  zh: "发现 %{hop_count} 跳和 %{flow_count} 个唯一 %{plural_flows}"
 unknown:
   en: "unknown"
   fr: "inconnu"
   tr: "bilinmeyen"
+  zh: "未知"
 icmp:
   en: "icmp"
   fr: "icmp"
   tr: "icmp"
+  zh: "icmp"
 udp:
   en: "udp"
   fr: "udp"
   tr: "udp"
+  zh: "udp"
 tcp:
   en: "tcp"
   fr: "tcp"
   tr: "tcp"
+  zh: "tcp"
 status_failures:
   en: "%{failure_count} of %{total_probes} (%{failure_rate}%) probes failed"
   fr: "%{failure_count} sur %{total_probes} (%{failure_rate}%) sondes ont échoué"
   tr: "%{failure_count} / %{total_probes} (%{failure_rate}%) sondajın başarısız olması"
+  zh: "%{failure_count} 个失败，共 %{total_probes} 个探测（%{failure_rate}%"
 status_failed:
   en: "Failed"
   fr: "Échec"
   tr: "Başarısız"
+  zh: "失败"
 status_running:
   en: "Running"
   fr: "En cours"
   tr: "Çalışıyor"
+  zh: "运行中"
 status_frozen:
   en: "Frozen"
   fr: "Gelé"
   tr: "Dondurulmuş"
+  zh: "冻结"
 awaiting_data:
   en: "Awaiting data..."
   fr: "En attente de données..."
   tr: "Veri bekleniyor..."
+  zh: "等待数据..."
```

Comment:
```suggestion
  zh: "等待数据……"
```

Reply:
Interesting, is `……` more common or correct in Chinese?

This is Unicode "Horizontal Ellipsis" (https://www.compart.com/en/unicode/U+2026) which comes from the “General Punctuation” Unicode block (https://www.compart.com/en/unicode/block/U+2000 ) .  I wonder how well supported this is on various terminals?

Reply:
`……` is the ellipsis in Chinese - as long as the terminal can show Chinese, it can show this as well.